csharp simplechat doesn't build

### Describe the bug
I followed the exact instructions for /samples/functions/csharp/simplechat

I have a Web PubSub resource created on my Azure account and configured it as specified in the instructions. I then copied the connection string into local.settings.json.

func extensions install does not install anything. it prints
```"No action performed because no functions in your app require extensions."```

func start does not build successfully. it prints with 5 errors..
```
Build FAILED.

/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/Functions.cs(58,28): error CS0103: The name 'MessageDataType' does not exist in the current context [/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/SimpleChat.csproj]
/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/Functions.cs(70,28): error CS0103: The name 'MessageDataType' does not exist in the current context [/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/SimpleChat.csproj]
/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/Functions.cs(83,28): error CS0103: The name 'MessageDataType' does not exist in the current context [/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/SimpleChat.csproj]
/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/Functions.cs(89,28): error CS0103: The name 'MessageDataType' does not exist in the current context [/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/SimpleChat.csproj]
/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/Functions.cs(102,28): error CS0103: The name 'MessageDataType' does not exist in the current context [/home/sgilfillan/Projects/playground-repos/azure-webpubsub/samples/functions/csharp/simplechat/SimpleChat.csproj]
    0 Warning(s)
    5 Error(s)
```

### To Reproduce
<!--
We ❤ code! Point us to a minimalistic repro project hosted in a GitHub repo.

We will close this issue if:
- the repro project you share with us is complex. We can't investigate custom projects, so don't point us to such, please.
- if we will not be able to repro the behavior you're reporting
-->

### Exceptions (if any)
<!-- 
Include the exception you get when facing this issue
-->

### Further technical details
- SDK and its version you use
I installed nrgok and ran `ngrok http 7071` and I have Azure Storage Emulator and Azure Functions Core Tools 3 both installed.

--- a/docs/references/functions-bindings.md
+++ b/docs/references/functions-bindings.md
@@ -38,7 +38,7 @@ Working with the trigger and bindings requires that you reference the appropriat
 Install the client library from [NuGet](https://www.nuget.org/) with specified package and version.
 
 ```bash
-func extensions install --package Microsoft.Azure.WebJobs.Extensions.WebPubSub --version 1.0.0-beta.1
+func extensions install --package Microsoft.Azure.WebJobs.Extensions.WebPubSub --version 1.0.0-beta.3
 ```
 
 [NuGet package]: https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.WebPubSub

--- a/samples/functions/csharp/notifications/notification.cs
+++ b/samples/functions/csharp/notifications/notification.cs
@@ -1,3 +1,4 @@
+using Azure.Messaging.WebPubSub;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.WebPubSub;
 using Microsoft.Extensions.Logging;

--- a/samples/functions/csharp/notifications/notifications.csproj
+++ b/samples/functions/csharp/notifications/notifications.csproj
@@ -4,7 +4,7 @@
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.3" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.7" />
   </ItemGroup>
   <ItemGroup>

--- a/samples/functions/csharp/simplechat/Functions.cs
+++ b/samples/functions/csharp/simplechat/Functions.cs
@@ -1,3 +1,4 @@
+using Azure.Messaging.WebPubSub;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;

--- a/samples/functions/csharp/simplechat/SimpleChat.csproj
+++ b/samples/functions/csharp/simplechat/SimpleChat.csproj
@@ -7,7 +7,7 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.12" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.11" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.3" />
   </ItemGroup>
   <ItemGroup>
     <None Update="host.json">

--- a/samples/functions/js/notifications/extensions.csproj
+++ b/samples/functions/js/notifications/extensions.csproj
@@ -5,7 +5,7 @@
 	<DefaultItemExcludes>**</DefaultItemExcludes>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.3" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.0.0" />
   </ItemGroup>
 </Project>

--- a/samples/functions/js/simplechat/extensions.csproj
+++ b/samples/functions/js/simplechat/extensions.csproj
@@ -5,7 +5,7 @@
 	<DefaultItemExcludes>**</DefaultItemExcludes>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0-beta.3" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.0.0" />
   </ItemGroup>
 </Project>

